gravitee-apim-console-webui/src/management/api/reporter-settings/reporter-settings-message/reporter-settings-message.component.html

```diff
@@ -15,172 +15,213 @@
+                <mat-error *ngIf="form.get('samplingValue').hasError('required')"> The sampling value is required. </mat-error>
+                <mat-error *ngIf="form.get('samplingValue').hasError('min')">
+                  The sampling value should be greater than {{ settings?.logging?.messageSampling?.count?.limit ?? 10 }}.
+                </mat-error>
+              </mat-form-field>
+            </ng-container>
+          }
+
+          @if (form.get('samplingType').value === 'TEMPORAL') {
+            <ng-container>
+              <p>Samples messages based on time duration.</p>
+              <p>
+                The value should conform to ISO-8601 duration format, e.g. PT1S for a duration of 1 second and be greater than
+                {{ settings?.logging?.messageSampling?.temporal?.limit ?? 'PT1S' }}.
+              </p>
+              <mat-form-field data-testid="sampling_value" class="settings__form-control__form-field">
+                <mat-label>Message sampling</mat-label>
+                <input matInput formControlName="samplingValue" />
+                <mat-error *ngIf="form.get('samplingValue').hasError('required')"> The sampling value is required. </mat-error>
+                <mat-error *ngIf="form.get('samplingValue').hasError('invalidISO8601Duration')">
+                  The sampling value should use ISO-8601 duration format, e.g.
+                  {{ settings?.logging?.messageSampling?.temporal?.limit ?? 'PT1S' }}.
+                </mat-error>
+                <mat-error *ngIf="form.get('samplingValue').hasError('minTemporal')">
+                  The sampling value should be greater than {{ settings?.logging?.messageSampling?.temporal?.limit ?? 'PT1S' }}.
+                </mat-error>
+              </mat-form-field>
+            </ng-container>
+          }
+
+          @if (form.get('samplingType').value === 'WINDOWED_COUNT') {
+            <ng-container>
+              <p>Samples a count of messages during a sliding duration window.</p>
+              <p>
+                Format is COUNT/DURATION (duration should conform to ISO-8601 duration format). The resulting rate must be lower than
+                {{ settings?.logging?.messageSampling?.windowedCount?.limit ?? '1/PT1S' }}.<br />
+                Example: <strong>2/PT15S</strong> will sample, at most, two messages during a 15-second sliding window.
+              </p>
+              <mat-form-field data-testid="sampling_value" class="settings__form-control__form-field">
+                <mat-label>Message sampling</mat-label>
+                <input matInput formControlName="samplingValue" />
+                <mat-error *ngIf="form.get('samplingValue').hasError('required')"> The sampling value is required. </mat-error>
+                <mat-error *ngIf="form.get('samplingValue').hasError('invalidFormat')">
+                  The sampling value follow this format: COUNT/DURATION, COUNT>0, DURATION is ISO-8601 duration format e.g.
```

Comment:
```suggestion
                  The sampling value must follow this format: COUNT/DURATION, COUNT>0, DURATION is ISO-8601 duration format e.g.
```
Or you can say The sampling value _should_ follow. Whichever you want.